RallyHereIntegration/Source/RallyHereIntegration/Private/RH_GameInstanceBootstrappers.cpp

```diff
@@ -507,7 +507,11 @@ void URH_GameInstanceServerBootstrapper::OnOSSLoginComplete(int32 ControllerId,
 	}
 	else
 	{
+#if RH_FROM_ENGINE_VERSION(5,2)
+		Identity->GetLinkedAccountAuthToken(ControllerId, FString(), IOnlineIdentity::FOnGetLinkedAccountAuthTokenCompleteDelegate::CreateUObject(this, &URH_GameInstanceServerBootstrapper::RetrieveOSSAuthTokenComplete));
```

Comment:
Interesting.  They just added another string argument to this between versions? (or maybe removed it?)

Reply:
The argument is a token type, for OSS's that have multiple (ex: Steam has App Tickets and Session Tickets).

In the only case that matters right now (Steam), we could pass in the type, but the steam OSS allows you to specify the default type already, so just using the empty string and the existing config works.  That does require a project level ini edit, currently, but that can be solved via documentation (and this change doesnt affect that, just silences the warning).